components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -947,6 +966,15 @@ - (CGRect)estimatedFrameForItemAtIndex:(NSUInteger)index {
   return CGRectMake(viewOriginX, 0, viewSize.width, viewSize.height);
 }
 
+- (CGPoint)contentOffsetNeededToCenterItemView:(UIView *)itemView {
+  CGFloat availableWidth = [self availableSizeForSubviewLayout].width;
+  CGFloat itemViewWidth = CGRectGetWidth(itemView.frame);
+  CGFloat contentOffsetX = CGRectGetMinX(itemView.frame) - (availableWidth - itemViewWidth) / 2.f;
```

Comment:
Can we make this `CGRectGetMinX(itemView.frame) - ((availableWidth - itemViewWidth) / 2.f);` just helps me understand the order of operations better or make it two statements, please.

Reply:
Sure, no problem

Reply:
Done.